tests/unit/test_pool.py

```diff
@@ -198,6 +213,63 @@ def test_get_non_expired(self):
             self.assertTrue(session._exists_checked)
             self.assertFalse(pool._sessions.full())
 
+    def test_spans_bind_get(self):
+        pool = self._make_one(size=4)
+        database = _Database("name")
+        SESSIONS = sorted([_Session(database) for i in range(0, 4)])
+        database._sessions.extend(SESSIONS)
+        pool.bind(database)
+
+        with trace_call("pool.Get", SESSIONS[0]) as span:
+            pool.get()
+            wantEventNames = [
+                "Acquiring session",
+                "Waiting for a session to become available",
+                "Acquired session",
+            ]
+            self.assertSpanEvents("pool.Get", wantEventNames, span)
+
+        # Check for the overall spans too.
+        self.assertSpanAttributes(
+            "pool.Get",
+            attributes=TestFixedSizePool.BASE_ATTRIBUTES,
+        )
+
+        wantEventNames = [
+            "Acquiring session",
+            "Waiting for a session to become available",
+            "Acquired session",
+        ]
+        self.assertSpanEvents("pool.Get", wantEventNames)
+
+    def test_spans_pool_bind(self):
+        pool = self._make_one(size=1)
+        database = _Database("name")
+        SESSIONS = []
+        database._sessions.extend(SESSIONS)
+        fauxSession = mock.Mock()
+        setattr(fauxSession, "_database", database)
+        try:
+            with trace_call("testBind", fauxSession):
+                pool.bind(database)
+        except Exception:
+            pass
+
+        wantEventNames = [
+            "Requesting 1 sessions",
+            "Creating 1 sessions",
+            "exception",
+            "exception",
```

Comment:
why are there 2 exception events?

Reply:
It is a quirk with opentelemetry-python; they add "exception" on every __exit__ call whether or not you set your own, I'll file a bug later on but for now I'll just remove our explicit invocation of "span.record_exception" and instead add our comment.